README.md

```diff
@@ -1,3 +1,67 @@
-# manticore-image-search
+# Demo: Image Search with Manticore Search
 
-Reverse image search using Manticore Search.
+**Publicly available demo - [image.manticoresearch.com](https://image.manticoresearch.com/)**
+
+[Blogpost about this project](https://manticoresearch.com/blog/reverse-image-search-demo/)
+
+## Just run the demo locally
+
+To run the project on your local machine, you need to have Docker installed with the compose plugin. Follow these commands to start:
+
+```
+git clone https://github.com/manticoresoftware/manticore-image-search.git
+cd manticore-image-search
+cd docker/containers/manticore
+
+wget https://github.com/manticoresoftware/manticore-image-search/releases/download/241204/backup.tar.gz && rm -fr backup && tar xzf backup.tar.gz
+
+cd ../..
+docker compose down -v
+docker compose up
+```
+
+After completing these steps, the project should be accessible at [http://localhost/](http://localhost/).
+
+## Further configuration
+
+The default port for the server is 80, so if you need to change it, update the `nginx` section in `app/config/app.ini.yml`.
+
+## Preparing for Deployment
+
+If you aim to use this project beyond a Manticore Search demo, such as an alternative to GitHub's issue search, there's a method for deploying it on a remote server. First, install [yoda](https://github.com/Muvon/yoda) on your machine and familiarize yourself with its documentation.
+
+#### Deployment
+
+For deployment, tweak the `docker/Envfile` with your server details and make sure you have passwordless authentication set up with your SSH key. Then, simply run:
+
+```bash
+yoda deploy --env=production
+```
+
+#### Setting Up a New Server in 5 Steps
```

Comment:
How is this title and section different from "Deployment"?

Reply:
We should configure server before we deploy to it, this is copy paste from what we have in GitHub issue search and just clarification how to follow steps due to 3rd party dependency